lib/tests/tests.c

```diff
@@ -3117,6 +3117,84 @@ compute_beta_coalescence_rate_fails(unsigned int num_ancestors, double alpha)
     return 0;
 }
 
+static void
+check_time_change(msp_t *msp, double t)
+{
+    const double tol = 1e-9;
+    double g, g2, t2;
+    g = msp->model.model_time_to_generations(&msp->model, t);
+    t2 = msp->model.generations_to_model_time(&msp->model, g);
+    CU_ASSERT_DOUBLE_EQUAL(t, t2, tol);
+    g2 = msp->model.generation_rate_to_model_rate(&msp->model, t);
+    CU_ASSERT_DOUBLE_EQUAL(g, g2, tol);
+}
+
+static void
+test_model_time_change_consistency1(double population_size, double t)
```

Comment:
The convention is to use the ``test_`` prefix for top-level tests (like in unittest), so let's change this to ``check_model_time_consistency``